### PR TITLE
Adding index comparison tool esdiff

### PR DESCRIPTION
**Pull request by schmichael**

### CODE REVIEW

File: cmd/esdiff/main.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
+package main
```
Comment (epsniff) [inline cmd/esdiff/main.go]: If you turn off replication and indexing on the target index until the copy has finished,  it should be at least twice as fast.

**Comment by kyledj** [comment]
:fist::fist::fist:
:fist::point_down::fist:
:fist::fist::fist:

File: cmd/esdiff/main.go (hunk 1 above)
Comment (epsniff) [inline cmd/esdiff/main.go]: ref: https://blog.codecentric.de/en/2014/05/elasticsearch-indexing-performance-cheatsheet/

**Comment by epsniff** [comment]
:+1:

File: cmd/esdiff/main.go (hunk 1 above)
Comment (schmichael) [inline cmd/esdiff/main.go]: awesome, made a ticket to hopefully add this in the future #4

**Pull request merged**